docs/section19/19-19.md

```diff
@@ -0,0 +1,111 @@
+            - Get the fan system object:  `fan_sys_b = hvac.fan_system`  
+            - Get list with the hvac system's fan system's supply, return, exhaust and relief fan powers in kW: `fan_sys_kW = list(get_fan_system_object_supply_return_exhaust_relief_total_kW_CFM(B_RMI,fan_sys_b)["Fan_Power"])`  
+            - Get list with the hvac system's fan system's supply, return, exhaust and relief fan CFM: `fan_sys_CFM = list(get_fan_system_object_supply_return_exhaust_relief_total_kW_CFM(B_RMI,fan_sys_b)["Fan_Power"]["Fan_CFM"])` 
+            - Get the total supply cfm, need to get the first element in the list of fan CFMs: `supply_cfm_b = fan_sys_CFM[0]` 
+            - Get the total exhaust cfm, need to get the third element in the list of fan CFMs: `total_exhaust_cfm = fan_sys_CFM[2]`   
+
+            - Check if the number of supply fans equals 1, if not go to rule assertion: `if list(get_fan_system_object_supply_return_exhaust_relief_total_kW_CFM(B_RMI,fan_sys_b)["Fan_Power"]["Fan_CFM"]["Fan_Qty"])[0] == 1:`  
+                - Set the more_than_one_supply_fan equal to false: `more_than_one_supply_fan = false`  
+                - Get the hvac system total supply fan kW, need to get the first element in the list of fan powers:`hvac_sys_total_supply_fan_power_b = fan_sys_kW[0]` 
+                - Get the hvac system total return fan kW, need to get the second element in the list of fan powers:`hvac_sys_total_return_fan_power_b = fan_sys_kW[1]` 
+                - Get the hvac system total exhaust fan kW, need to get the third element in the list of fan powers:`hvac_sys_total_exhaust_fan_power_b = fan_sys_kW[2]` 
+                - Get the hvac system total relief fan kW, need to get the fourth element in the list of fan powers:`hvac_sys_total_relief_fan_power_b = fan_sys_kW[3]`     
+
+                - Calculate the total modeled fan power for the hvac system in the baseline: `total_fan_power = hvac_sys_total_supply_fan_power_b + hvac_sys_total_return_fan_power_b + hvac_sys_total_exhaust_fan_power_b + hvac_sys_total_relief_fan_power_b`  
+
+                - Cycle through each zone associated with the HVAC system to get the exhaust fan power: `for zone_b in zone_list:`  
+                    - Create a zone object from the zone id (not sure how the RCT will do this): `zone_b_obj = zone_b`  
+                    - Check if there are any exhaust fan objects associated with the zone: `if zone_b_obj.zonal_exhaust_fan != Null:`  
+                        - Get the fan power: `fan_elec_power = get_fan_object_electric_power(B_RMI,zone_b_obj.zonal_exhaust_fan)`    
+                        - Add to the total power: `total_fan_power = total_fan_power + fan_elec_power`                    
+                
+                Below determines the expected fan power:
+                - Check if there are any energy recovery objects associated with the fan system object: `if fan_sys_b.air_energy_recovery != Null:` 
```

Comment:
This will likely need revisions after we modify the energy recovery data elements in the schema.